Neos.Neos/Classes/Fusion/Helper/CachingHelper.php

```diff
@@ -70,15 +71,46 @@ public function nodeTag($nodes)
     /**
      * Generate an `@cache` entry tag for a node type
      * A cache entry with this tag will be flushed whenever a node
-     * (for any variant) that is of the given node type (including inheritance)
-     * is updated.
+     * (for any variant) that is of the given node type(s)
+     * (including inheritance) is updated.
      *
-     * @param NodeType $nodeType
+     * @param string|NodeType|string[]|NodeType[] $nodeType
+     * @return string[]
+     */
+    public function nodeTypeTag($nodeType) : array
+    {
+        if (!is_array($nodeType) && !($nodeType instanceof \Traversable)) {
+            $result = [$this->getNodeTypeTagFor($nodeType)];
+            return array_filter($result);
```

Comment:
array_filter filters everything falsy if no callback is given, which is true for empty string.